WordPress/gutenberg
# List all the font families and font sizes from all the theme.json origins in the font picker

## What?
List all the font families from all the theme.json origins in the font picker

## Why?

Currently, the font family picker is listing only font families from custom theme.json origin (global styles) if it's defined.

## How?
Concat all the font families available from all the origins.

## Testing Instructions
1. Add font families definitions to `settings.typography.fontFamilies` in the custom global styles. (The theme.json is stored in the database)
2. Check that in the typography panel the font picker includes the font families from custom and from theme origins.


Fixes: https://github.com/WordPress/gutenberg/issues/51487
